exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Queries.kt

```diff
@@ -499,6 +499,26 @@ fun <T : Table> T.updateReturning(
     return ReturningStatement(this, returning, update)
 }
 
+fun <T : Table, E> T.batchUpdate(
+    data: Iterable<E>,
+    limit: Int? = null,
```

Comment:
I don't expect that standard update statment supports `limit` buy I may be wrong here.